recipes/py3dep/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "py3dep" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cheginit/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 3fd0644c74d16c875136232897a74d673dba835d866b8b6c9349f2efc7c4eda3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools_scm
+  run:
+    - python >=3.6
+    - pip
```

Comment:
Is `pip` needed to run this package?

Reply:
No, I don't need it. I just kept it since I've seen other packages do.

Reply:
@synapticarbors I just removed `pip` and it passed the checked. Thanks!